translations/de.ts

```diff
@@ -1340,7 +1344,7 @@ will be sent to them when they appear online to you.</source>
     <message>
         <location filename="../src/widget/form/genericchatform.cpp" line="515"/>
         <source>Send file(s)</source>
-        <translation>Datei senden</translation>
+        <translation>Datei schicken</translation>
```

Comment:
Soweit ich weiß verwenden mittlerweile alle Mailprogramme "senden". Mir is es ehrlich gesagt wichtig, dass man uns (die qTox/Tox Community) versteht. :)